tests/InputFiles/CurvedScalarWave/ScalarWaveKerrSchild.yaml

```diff
@@ -0,0 +1,100 @@
+# Distributed under the MIT License.
```

Comment:
I don't love the idea of adding another input file test (right now) because it means it's another executable we need to compile during CI. I think once we have initial data being controlled from the input file (hopefully ~1 month time scale) then this won't be an issue anymore. Maybe we omit for now and add once the initial data change is done?